packages/wpcom-checkout/src/field.tsx

```diff
@@ -61,7 +64,8 @@ export default function Field( {
 				</Label>
 			) }
 
-			<InputWrapper>
+			<InputWrapper isError={ isError }>
+				{ vatDetails && <span className="vat-field__overlay-prefix">{ vatDetails?.country }</span> }
```

Comment:
Can we make this something more generic so it could be more easily used for non-VAT-related purposes? Right now it's awfully specific and this component could be used for all kinds of fields.

How about... `prefix`? You could make its type `ReactNode` (like `errorMessage`) and then change this to:

```suggestion
				{ prefix && <span className="field__overlay-prefix">{ prefix }</span> }
```

Reply:
Good idea, resolved.